src/sass/style.scss

```diff
@@ -64,3 +64,31 @@ body {
 .grav-c-two-columns-text {
   margin: 2rem auto 0em auto;
 }
+
+// Temporary overrides - About page partner logos
+.grav-c-list-inline-row {
+  // Negative margin on container to cancel out left margin of left-most logo(s)
+  margin-left: -1 * $bigger-gap-size;
```

Comment:
The `$bigger-gap-size` variable name was just a dummy example. This doesn't compile since that's not defined anywhere.

Please replace it with whatever your desired horizontal gap between logos is. For what it's worth `$grav-sp-l` ought to match your original layout since it's twice the size of `$grav-sp-m`.